pandas/core/frame.py

```diff
@@ -10345,6 +10346,15 @@ def apply(
             Pass keyword arguments to the engine.
             This is currently only used by the numba engine,
             see the documentation for the engine argument for more information.
+
+        jit : function, optional
```

Comment:
what if the 3rd party implementation isn't a jit?  e.g. it is just parallel?

Reply:
Would you rename the parameter name to for example `executor`? I'm happy with that. I guess that would make naming more accurate if this is used for other use cases as running in parallel, which is possible with this interface. Do you have a specific use case in mind?

Reply:
id just use 'engine', avoid having multiple similar keywords

Reply:
Cool, that makes sense, thanka for the feedback. This is how it's implemented now in the PR, just using the existing parameter `engine`.